stories/components/feedback/donut-chart.stories.tsx

```diff
@@ -0,0 +1,389 @@
+            component="Select"
+            items={[
+              {
+                label: "Tooltip data source",
+                defaultValue: "all",
+                items: [
+                  { label: "all", value: "all" },
+                  { label: "segment", value: "segment" },
+                ],
+                value: props.tooltipDataSource,
+                onChange: (value) => {
+                  setProps((prev) => ({
+                    ...prev,
+                    tooltipDataSource: value as TooltipDataSourceType,
+                  }))
+                },
+              },
+            ]}
+          />
+
+          <PropControl
+            component="Switch"
+            items={[
+              {
+                label: "tooltip",
+                isChecked: props.withTooltip,
+                onChange: () =>
+                  setProps((prev) => ({
+                    ...prev,
+                    withTooltip: !prev.withTooltip,
+                  })),
+              },
+              {
+                label: "legend",
+                isChecked: props.withLegend,
+                onChange: () =>
+                  setProps((prev) => ({
+                    ...prev,
+                    withLegend: !prev.withLegend,
+                  })),
+              },
+              {
+                label: "label",
+                isChecked: props.withLabel,
+                onChange: () =>
+                  setProps((prev) => ({
+                    ...prev,
+                    withLabel: !prev.withLabel,
+                  })),
+              },
+              {
+                label: "label line",
+                isChecked: props.withLabelsLine,
+                onChange: () =>
+                  setProps((prev) => ({
+                    ...prev,
+                    withLabelsLine: !prev.withLabelsLine,
+                  })),
+              },
+            ]}
+          />
+        </VStack>
+      </Wrap>
+    </Wrap>
+  )
+}
+
+export const withSize: Story = () => {
+  const data: CellProps[] = useMemo(
+    () => [
+      { name: "USA", value: 400, color: "red.500" },
+      { name: "India", value: 300, color: "orange.500" },
+      { name: "Japan", value: 100, color: "blue.500" },
+      { name: "Other", value: 200, color: "gray.500" },
+    ],
+    [],
+  )
+
+  return (
+    <>
+      <DonutChart data={data} size="sm" />
+      <DonutChart data={data} size="md" />
+      <DonutChart data={data} size="lg" />
+      <DonutChart data={data} size="full" />
+    </>
+  )
+}
+
+export const withLegend: Story = () => {
+  const data: CellProps[] = useMemo(
+    () => [
+      { name: "USA", value: 400, color: "red.500" },
+      { name: "India", value: 300, color: "orange.500" },
+      { name: "Japan", value: 100, color: "blue.500" },
+      { name: "Other", value: 200, color: "gray.500" },
+    ],
+    [],
+  )
+
+  return <DonutChart data={data} withLegend />
+}
+
+export const withValueFomatter: Story = () => {
```

Comment:
Change `withValueFomatter` to `withValueFormatter`.

Reply:
Done.